_community/maykinmedia.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: Association 1
-subtitle: Association 1 tagline here
+title: Maykin Media B.V.
+subtitle: Developer
 description: This is a association description
```

Comment:
Should association be changed to vendor?

Reply:
done